### DISCUSSION TITLE
Remove Chiaki Cask

### DISCUSSION THREAD

**Discussion by iMonZ** [Homebrew/discussions#3764] [Casks]
A good point is removing the Chiaki Cask https://github.com/Homebrew/homebrew-cask/blob/master/Casks/chiaki.rb

Since the last version build is two years ago and doesn't work on the latest macOS version.

The Owner of this repo is still committing regularly, so it's much better creating a formula of this instead of a cask.

**Comment by SMillerDev** [comment]
But formulae also need frequent releases. It's much better to just ask the developer to make a new release.

**Reply by iMonZ** [reply]
Actually on the formula you can mark the commit as the version.
And also the cask Is not signed

**Reply by SMillerDev** [reply]
Not in official formulae you can't: https://docs.brew.sh/Acceptable-Formulae#stable-versions
